definitions/scenarios/upgrade_to_satellite_6_2.rb

```diff
@@ -43,7 +43,7 @@ class Migrations < Abstract
 
     def compose
       add_step(Procedures::Repositories::Setup.new(:version => '6.2'))
-      add_step(Procedures::Packages::Update.new(:assumeyes => false))
+      add_step(Procedures::Packages::Update.new(:assumeyes => assumeyes?))
```

Comment:
If you don't pass param `assumeyes` to `Procedures::Packages::Update` procedure then nil case is already handled in file [update.rb](https://github.com/theforeman/foreman_maintain/blob/master/definitions/procedures/packages/update.rb#L9)  where it takes value from `assumeyes?` method.
So no need to pass param `assumeyes` here.

Reply:
@kgaikwad  Thanks for reviewing.
Right, but the instance variable in [update.rb](https://github.com/theforeman/foreman_maintain/blob/master/definitions/procedures/packages/update.rb#L9) is different than the one used here.

Arg `:assumeyes => assumeyes?` in this case refers to [Scenario#assumeyes?](https://github.com/swapab/foreman_maintain/blob/21208-package-update-comply-assumeyes/lib/foreman_maintain/scenario.rb#L10).
So, the argument is not optional it needs to be passed.

Reply:
Thinking more about this, I would probably not even ask the user and put always true here: they already confirmed they want to update before and there is a little chance hey would ever answer no here.